modules/tinymce/src/themes/silver/test/ts/browser/editor/color/ColorPickerSanityTest.ts

```diff
@@ -52,14 +67,27 @@ describe('browser.tinymce.themes.silver.editor.color.ColorPickerSanityTest', ()
       const setHexWhite = setHex('ffffff');
       const setHexBlack = setHex('000000');
 
-      const pSubmitDialog = async (editor: Editor) => {
+      const submit = async (editor: Editor) => {
```

Comment:
This needs to be called `pSubmit` as it's an async function.

Reply:
Maybe `pSubmitInner` or `pSubmitRaw` to make it clear that this shouldn't be called from directly within a test / that it's an implementation detail for the main functions